example.md

```diff
@@ -80,6 +80,7 @@ This is the example of an ordered list in our documentation.
 
 ### With Icons
 
+* {:.flag} This is a flag icon
```

Comment:
Please describe here or in the wiki the usecase for the new icon. In what case it should be used. We had problems with different usages of same list icon in the past.

Reply:
Good question. @hguthrie, do you have any use cases for this that you were thinking of?

Reply:
The reason I suggested having a class is so that it is easier for a writer to draw attention to something new. I did not have a bullet in mind. I would want to draw attention to something in a menu or navigation, perhaps.
But, for a bullet, I suppose it would be useful on a landing page that contains a standing list of assets, or links to guides or topics.

Reply:
@hguthrie, we have a + icon for a new feature. Perhaps flag should be used to call out some important features? Or the ones that introduce significant changes?

Reply:
https://jira.corp.magento.com/browse/MAGEDOC-3136

Reply:
@hguthrie, please don't use links to internal resources in public repositories.